### PR TITLE
Send ITTI message from PCEF create session callback

### PR DESCRIPTION
**Pull request by ssanadhya**
## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

File: lte/gateway/c/oai/tasks/sgw/pgw_handlers.c [hunk 1]
```diff
@@ -184,6 +184,47 @@ void _spgw_handle_s5_response_with_error(
...
+  s5_response.context_teid                 = pcef_csr_resp_p->teid;
+  s5_response.eps_bearer_id                = pcef_csr_resp_p->eps_bearer_id;
+  s5_response.status                       = pcef_csr_resp_p->sgi_status;
+  s5_response.failure_cause                = S5_OK;
+
+  s_plus_p_gw_eps_bearer_context_information_t* bearer_ctxt_info_p =
+      sgw_cm_get_spgw_context(pcef_csr_resp_p->teid);
+
+  sgw_eps_bearer_ctxt_t* eps_bearer_ctx_p =
+      bearer_ctxt_info_p->sgw_eps_bearer_context_information.pdn_connection
```
Comment (ssanadhya) [inline lte/gateway/c/oai/tasks/sgw/pgw_handlers.c:204]: Use `sgw_cm_get_eps_bearer_entry` here

File: lte/gateway/c/oai/tasks/sgw/spgw_state.cpp [hunk 2]
```diff
@@ -42,6 +42,8 @@ spgw_state_t* get_spgw_state(bool read_from_db) {
 }
 
 hash_table_ts_t* get_spgw_ue_state() {
+  OAILOG_DEBUG(
+      LOG_SPGW_APP, "get_spgw_ue_state called by thread id %u", pthread_self());
```
Comment (talkhasib) [inline lte/gateway/c/oai/tasks/sgw/spgw_state.cpp:46]: You can compare against and only log (maybe a warning) when there is a mismatch 
itti_desc.threads[TASK_GET_THREAD_ID(TASK_SPGW_APP)].task_thread

File: lte/gateway/c/oai/tasks/sgw/sgw_task.c [hunk 3]
```diff
@@ -238,14 +244,6 @@ int spgw_app_init(spgw_config_t* spgw_config_pP, bool persist_state) {
...
   }
 
-  FILE* fp         = NULL;
-  bstring filename = bformat("/tmp/spgw_%d.status", g_pid);
-  fp               = fopen(bdata(filename), "w+");
-  bdestroy_wrapper(&filename);
-  fprintf(fp, "STARTED\n");
-  fflush(fp);
-  fclose(fp);
-
```
Comment (talkhasib) [inline lte/gateway/c/oai/tasks/sgw/sgw_task.c:248]: What was this really for?

Comment (ssanadhya) [reply lte/gateway/c/oai/tasks/sgw/sgw_task.c:236]: This was legacy code which was not being used anywhere, except create a lot of files in `/tmp/`

**Pull request closed without merging**